### ISSUE TITLE
Webp images are blocked by .htaccess

### ISSUE THREAD

**Issue by grzegorzgit** [PrestaShop/PrestaShop#24092]
Hi,
please remove blocking of webp images from /img folder. Currently .htaccess allows only specific types of files to be accessed from that folder, and webp is not one of them. 

I understand that webp is relatively new format, but Lighthouse reports using .jpg and .png as an issue, and suggests replacing it with webp format. I am not asking about changing PrestaShop images to webp, I believe for a long time this will be to be done by each PS user separately, but at least add the extension to .htaccess so people won't lose time -like me- on finding out why images are not showing up.

It is not the genral .htaccess, but the one inside /img folder.

Btw. it would be also desired to allow svg format for uploading logo.

#### Steps to Reproduce 

Put image with .webp extension into /img folder.
Try to display the image on a template using `<img src="/img/image.webp" />`, or just open the path in browser.

PrestaShop version: 1.7.7.2

**Comment by florine2623** [comment]
Hello @grzegorzgit ,

I added a webp image to my folder /img, but I can't access it through _www.my_website.com/img/cat.webp_
[cat.webp.zip](https://github.com/PrestaShop/PrestaShop/files/6326197/cat.webp.zip)
![Capture d’écran 2021-04-16 à 17 24 22](https://user-images.githubusercontent.com/16019289/115047192-98495780-9ed8-11eb-91a4-a748876b2ac4.png)

With a JPG, PNG or GIF, it work displays.

I'll ping PrestaShop product team to see what they think !

Please stay tuned.
Thanks!

**Comment by florine2623** [comment]
Ping @PrestaShop/product-team ,
What do you think about this suggestion ?

**Comment by marionf** [comment]
Duplicate of #10356

**Comment by marionf** [comment]
Hello,

This issue is a duplicate of #10356
Please follow this issue to stay informed

Thank you!
